tests/contrib/test_viz.py

```diff
@@ -67,6 +68,10 @@ def test_plot_results(datadir):
 
 
 @pytest.mark.mpl_image_compare
+@pytest.mark.xfail(
```

Comment:
I'm going with `xfail` over `skip` here as skip just doesn't run the test at all, but `xfail` will guarantee that the behavior is failing as expected and so is a stronger test.